app/(api)/_datalib/media/createMediaItem.ts

```diff
@@ -3,17 +3,20 @@ import { ObjectId } from 'mongodb';
 
 import { getDatabase } from '../../_utils/mongodb/mongoClient.mjs';
 import { HttpError, NoContentError } from '../../_utils/response/Errors';
+import { media_collection } from '@utils/constants/media';
```

Comment:
I think this can just not be a constant. Also, probably don't put constants in @utils. @utils is for utilities. I think you can just hardcode 'media' or smth.